6.0.541.json

```diff
@@ -0,0 +1,10 @@
+{
+  "microsoft.net.sdk.android": "32.0.476",
```

Comment:
in order to use `tizen` do we need to update all sdks?

Reply:
it is a maui release 6.0.541. so I think we need update them all

Reply:
That will be a breaking change? IF so should this version be 2.0?

Reply:
we'll break users who still use maui 6.0.312, but because it is the last version based on .NET 6, and it only adds Tizen support, I don't think we need to upgrade to version 2.0

Reply:
on the other hand, what doesn't work if we stay on 6.0.312? can we upgrade to 6.0.400?

Reply:
@VladislavAntonyuk 
6.0.300 (dotnet band 300) is not support MAUI 6.0.541 so we need to upgrade to 6.0.400
and Tizen backend is changed at [6.0.540](https://github.com/dotnet/maui/releases/tag/6.0.540) so we have to use 400 band

https://www.nuget.org/packages/Microsoft.NET.Sdk.Maui.Manifest-6.0.300
It is 300 band Maui sdks

https://www.nuget.org/packages/Microsoft.NET.Sdk.Maui.Manifest-6.0.400
It is 400 band Maui sdks

Reply:
Why do we need MAUI 6.0.541. We want to keep MAUI 6.0.312

Reply:
If application compiles with MAUI 6.0.312 we have to keep it. It’s up to user if they want to upgrade and make Tizen work in runtime

Reply:
Ok, but to use tizen, we have to use maui version higher than 6.0.540, so this PR should be postpone until CommunityToolkit.Maui is using higher than 6.0.540.
or I should re-target to net7.0

Reply:
Why do you need 6.0.540? I think Tizen was added earlier

Reply:
> Why do you need 6.0.540? I think Tizen was added earlier

TIzen was officially included from .NET MAUI RC2 (6.0.300-rc.2), but there was a change in underlying platform UI Framework in .NET MAUI Service Release 5 (6.0.540). Please refer to https://github.com/dotnet/maui/pull/9620.